tests/OmniSharp.Roslyn.CSharp.Tests/CompletionFacts.cs

```diff
@@ -270,12 +270,34 @@ public static void Test(this object o)
 
             Assert.Single(resolved.Item.AdditionalTextEdits);
             var additionalEdit = resolved.Item.AdditionalTextEdits[0];
-            Assert.Equal(NormalizeNewlines("using N2;\n\nnamespace N1\r\n{\r\n    public class C1\r\n    {\r\n        public void M(object o)\r\n        {\r\n            o"),
+            Assert.Equal(NormalizeNewlines("using N2;\n\n"),
```

Comment:
@bjorkstromm @NTaylorMullen these change will, I think, make import completion easier to support from Cake/Razor, as we're not sending the whole document anymore. Just the import changes.